config/config.exs

```diff
@@ -26,7 +26,12 @@ end
 
 
 config :smart_home_firmware,
-  hub: "192.168.0.14:4000"
+  hub: "192.168.0.14:4000",
+  ssl: false
```

Comment:
Can we figure out how to do SSL/TLS on the _home_ network? (and can you get it working on your "room" network?)

Reply:
Its *possible* - i'll open an issue as its out of scope for this PR